modules/UI/videolayout/Filmstrip.js

```diff
@@ -247,11 +255,11 @@ const Filmstrip = {
      * @returns {{availableWidth: number, availableHeight: number}}
      */
     calculateAvailableSize() {
-        let availableHeight = interfaceConfig.FILM_STRIP_MAX_HEIGHT;
-        const thumbs = this.getThumbs(true);
-        const numvids = thumbs.remoteThumbs.length;
-
-        const localVideoContainer = $('#localVideoContainer');
+        const state = APP.store.getState();
+        const currentLayout = getCurrentLayout(state);
+        const isHorizontalFilmstripView
+            = currentLayout === LAYOUTS.HORIZONTAL_FILMSTRIP_VIEW;
+        const isTileView = currentLayout === LAYOUTS.TILE_VIEW;
```

Comment:
Maybe I could just do this with percentages.

Reply:
I've pushed a new implementation in a separate commit, which essentially tries to ignore legacy filmstrip resizing logic and abuses !important to set widths/heights using css.